tools/tutorial_vehicle_signals/speed-consumer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.12-slim-bookworm
```

Comment:
I think this image is ok. But if you want to have an even smaller image (because it is designed for fast demonstration/tutorial purposes), the image `python:3.12.1-alpine3.18` is around 60% smaller.

Reply:
I think the image size is not that important for the tutorial but thanks for the hint.